packages/apps/docs/src/components/Layout/components/TitleHeader/TitleHeader.tsx

```diff
@@ -15,22 +20,22 @@ export const TitleHeader: FC<IProps> = ({ title, subTitle, icon }) => {
   const Icon = icon ? ProductIcon[icon] : null;
 
   return (
-    <HeaderWrapper data-cy="titleheader">
-      <StyledHeader>
-        <Wrapper>
+    <div data-cy="titleheader" className={headerWrapperClass}>
+      <header className={headerClass}>
+        <div className={wrapperClass}>
           <Stack alignItems="center">
             {Icon && <Icon size="heroHeader" />}
             <Heading as="h1">{title}</Heading>
           </Stack>
           {subTitle !== undefined && (
-            <SubHeader>
-              <Heading as="h5" bold={false}>
+            <span className={subheaderClass}>
+              <Heading as="h5" variant="h6" bold={false}>
```

Comment:
```suggestion
              <Heading as="h5" variant="h6" bold={false}>
```
this looks weird, h5 with h6